tools/gh-rename-branches.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+
+echo "This script is set to rename the default branch only on test repositories by default. Please open the file and \
+insert the correct variable name - REPOSITORIES - in the for loop to run the rename script."
+echo -e "If the default branch is not master and no master branch exists, \
+no renaming will happen in that particular repository. \n"
+
+echo -e "\033[1mWhat is your GitHub personal access token?\033[0m See here for more: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token >"
+
+read usertoken
```

Comment:
Might want to add `-r` and `-s` options.

`-r` avoids problems if for some reason there's a backslash. There shouldn't be, but if there is we probably want a literal backslash.

`-s` avoids printing the token to the screen when it's entered.